### ISSUE TITLE
Why is pytest a dependency of this package

### ISSUE THREAD

**Issue by DManowitz** [conda-forge/petl-feedstock#35]
### Comment:

According to the [setup.py](https://github.com/petl-developers/petl/blob/v1.7.12/setup.py) file for this package in its github repo, pytest does not appear to be a dependency of this package, so why is it included as a dependency of this conda-forge package?

**Comment by juarezr** [comment]
`pytest` is a dev dependency of `petl` package.

**Comment by DManowitz** [comment]
Yes, but it is a **dev** dependency, not an **install** dependency, so it seems to me that it should not be included as a dependency on conda-forge.  Those that want to develop this package should be getting it directly from github.

**Comment by juarezr** [comment]
Good catch!

Fix [on the works](https://github.com/juarezr/petl-feedstock/pull/2)

**Comment by juarezr** [comment]
@conda-forge-admin, please rerender

**Comment by juarezr** [comment]
@conda-forge-admin, please rerender

**Comment by juarezr** [comment]
@conda-forge-admin, please update for conda-build 3
